### DISCUSSION TITLE
Features /nightvision command

### DISCUSSION THREAD

**Discussion by MrTokyo12345** [EssentialsX/Essentials#4579] [Ideas and Feature Suggestions] [answered]
Hi, you can't add the comand (/nightvision on/off)

If you activate this command, it gives you the infinite night vision.

**Comment by Chew** [comment, marked as answer]
This was discussed (and rejected) in https://github.com/EssentialsX/Essentials/issues/3825. 

General reason was, we already have a Minecraft command for it, and you can simply alias it:

> There's no reason why this should be implemented in Essentials when the vanilla command also does what you want, and you can alias each effect in however way you want via commands.yml.
